R/get.R

```diff
@@ -336,13 +339,13 @@ spod_get_zones_v2 <- function(
     }
     if (isFALSE(quiet)) {
       message("Downloading missing zones data...")
-      curl::multi_download(
-        urls = metadata_zones_for_download$target_url,
-        destfiles = metadata_zones_for_download$local_path,
-        resume = TRUE,
-        progress = TRUE
-      )
     }
+    curl::multi_download(
```

Comment:
@Robinlovelace that was what failing. the files were not being downloaded if quiet=TRUE, which it was when called from within spod_get. I did not experience this because because I had these files cached long before. Stupid mistake...